### DISCUSSION TITLE
遇到了Cannot use chat template functions because tokenizer.chat_template is not set and no template argument was passed!的问题

### DISCUSSION THREAD

**Discussion by WCSY-YG** [QwenLM/Qwen3#1053] [General]
我正在使用官方示例的代码，结果遇到了这个错误，请问这个是什么问题，怎么解决呢
![image](https://github.com/user-attachments/assets/f784656a-5f8f-439e-a285-0d5668528344)

**Comment by jklj077** [comment]
看下./Qwen/Qwen2.5-7B-Instruct目录里的内容，再就是tokenizer_config.json的内容

**Reply by WCSY-YG** [reply]
谢谢，已经解决了，从hf镜像站下载的模型，有点问题
